src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformerTest.php

```diff
@@ -307,15 +307,15 @@ public function testReverseTransformWrapsIntlErrors()
     }
 
     /**
-     * @expectedException \Symfony\Component\Form\Exception\UnexpectedTypeException
+     * @expectedException \TypeError
      */
     public function testValidateDateFormatOption()
```

Comment:
those tests are useless now as they only test php internal behavior

Reply:
I'm sure there are more left over, PR is so big...
Additional PR welcomed :)